### ISSUE TITLE
template error after importing from blogspot

### ISSUE THREAD

**Issue by shadow-identity** [django-blog-zinnia/blogger2zinnia#1]
Hi, importing was without any errors, but I can't load any blog page:

```
NoReverseMatch at /blog/
Reverse for 'entry_detail' with arguments '()' and keyword arguments '{'year': '2007', 'slug': u'', 'day': '16', 'month': '12'}' not found. 1 pattern(s) tried: [u'blog/(?P<year>\\d{4})/(?P<month>\\d{2})/(?P<day>\\d{2})/(?P<slug>[-\\w]+)/$']
```

the error is here:
`<a href="{{ object.get_absolute_url }}" title="{{ object.title }}" rel="bookmark">`

It is because the slugs of records written with Cyrillic symbols are just empty. It is actual for tags too. So cyrillic is just dropped.

**Comment by Fantomas42** [comment]
Okay I will take a look on this.

**Comment by Fantomas42** [comment]
I have published on Pypi a new release (v1.1) of the package fixing the issue.

Note for the tags, I don't see a solution.
